.eslintrc.js

```diff
@@ -1,14 +1,14 @@
 module.exports = {
   root: true,
   extends: [
-    '@react-native',
```

Comment:
Is this removal intentional, or happened by mistake during rebase? This is the new ESLint config.